<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QWidget" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QLabel" name="labelRecordSpeed">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Record Speed (ms):</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="recordSpeedSpinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>99</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelVisualizationSpeed">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualization Speed (ms):</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="visualizationSpeedSpinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>99</number>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
